var/spack/repos/builtin/packages/ants/package.py

```diff
@@ -15,13 +15,26 @@ class Ants(CMakePackage):
        processing library to which ANTs developers contribute.
     """
 
-    homepage = "http://stnava.github.io/ANTs/"
+    homepage = "https://stnava.github.io/ANTs/"
     url      = "https://github.com/ANTsX/ANTs/archive/v2.2.0.tar.gz"
 
     version('2.2.0', sha256='62f8f9ae141cb45025f4bb59277c053acf658d4a3ba868c9e0f609af72e66b4a')
+    version('2.3.5', sha256='2fddfd5f274a47f1c383e734a7e763b627c4a8383d2d3b9971561f335016bb0a')
 
     depends_on('zlib', type='link')
 
+    variant('MINC', values=bool, default=True,
+            description='Build ITK with MINC support')
```

Comment:
We usually use lowercase for variant names and variants are Boolean by default:
```suggestion
    variant('minc', default=True, description='Build ITK with MINC support')
```